src/char/int_guild.cpp

```diff
@@ -499,6 +499,7 @@ std::shared_ptr<CharGuild> inter_guild_fromsql( int32 guild_id ){
 		Sql_GetData(sql_handle, 0, &data, nullptr); e->account_id = atoi(data);
```

Comment:
@Lemongrass3110 account_id can be changed to strtoul(data, nullptr, 10); too?

Reply:
No lets keep it like this right now, so that I can merge this PR.